.github/workflows/scripts/linux/scons-build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh -ve
+
+CCFLAGS=-fdiagnostics-color=always scons
```

Comment:
Since this is now in a separate file, I think it should go on two lines:

```suggestion
export CCFLAGS=-fdiagnostics-color=always
scons
```

I'm _pretty_ sure that would have the same effect.